### PR TITLE
Iamsamirzon patch 5

### PR DESCRIPTION
**Pull request by iamsamirzon**
This PR is to update the README.MD file to refer the system release notes of Alpha-1 and Alpha-2 release of NV2 project. Also included is a new file for the Alpha-2 release notes

### CODE REVIEW

File: RELEASENOTES/v2.0.0.alpha-2.MD [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Notary V2 Alpha 2 Release
+
+## Overview
+NotaryV2 working group is pleased to announce the second release, Alpha 2. This release updates the [notaryproject/notaryproject](https://github.com/notaryproject/notaryproject) repo to provide the community with updated specifications for NV2. The refrence implementation of Notation client for signing and verifying artifacts remains unchanged.
```
Comment (gokarnm) [inline RELEASENOTES/v2.0.0.alpha-2.MD:4]:
```suggestion
NotaryV2 working group is pleased to announce the second release, Alpha 2. This release updates the [notaryproject/notaryproject](https://github.com/notaryproject/notaryproject) repo to provide the community with updated specifications for NV2. The refrence implementation of Notation client for signing and verifying artifacts remains unchanged. 
```

File: RELEASENOTES/v2.0.0.alpha-2.MD [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+### Notary V2 specifications
+Defined Trust Policy, Trust Store,  Signature Verifiction and Signing workflows along with extensibility mechanisms via Notation plugins.
+
+### Notation CLI Client
+Notation is a standalone client and is not integrated with Docker or Containerd clients. It uses a fixed configuration scheme in this release, where the certificates for signing and verifying are kept in a file. Refer https://github.com/notaryproject/notation/blob/main/specs/notation-config.md. It supports the operations for signing and verifying artifacts. This release depends on [oras-artifacts-spec](https://github.com/oras-project/artifacts-spec/) registry support for storing signatures. See [getting started](https://github.com/notaryproject/notation/blob/main/docs/hello-signing.md#getting-started) for using a local CNCF Distribution instance. 
+
+To start using Notation Refer: https://github.com/notaryproject/notation/blob/main/hello-signing.md
+
+### Notation Library
+Written in the GO language the Library is currently focused on enabling the Notation CLI.
```
Comment (gokarnm) [inline RELEASENOTES/v2.0.0.alpha-2.MD:27]:
```suggestion
Written in the Go language the Library is currently focused on enabling the Notation CLI.
```

File: RELEASENOTES/v2.0.0.alpha-2.MD [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+## Goal of the release
+- Update the NotaryV2 specifications to allow implementors to add the next set of capabilties for notation client and library.
+
+*This is not a production ready release. Recommended for use in non-production environments only*
+
+## List of feature/capabilities added with this release
+For detailed notes on each of the individual repositories (NotaryProject, Notation and Notation-go-lib) users can refer to the individual release notes. A summary view is provided in this document.
+
+### Notary V2 specifications
+Defined Trust Policy, Trust Store,  Signature Verifiction and Signing workflows along with extensibility mechanisms via Notation plugins.
```
Comment (NiazFK) [inline RELEASENOTES/v2.0.0.alpha-2.MD:19]: (sp) Verification

File: RELEASENOTES/v2.0.0.alpha-2.MD [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+### Notation CLI Client
+Notation is a standalone client and is not integrated with Docker or Containerd clients. It uses a fixed configuration scheme in this release, where the certificates for signing and verifying are kept in a file. Refer https://github.com/notaryproject/notation/blob/main/specs/notation-config.md. It supports the operations for signing and verifying artifacts. This release depends on [oras-artifacts-spec](https://github.com/oras-project/artifacts-spec/) registry support for storing signatures. See [getting started](https://github.com/notaryproject/notation/blob/main/docs/hello-signing.md#getting-started) for using a local CNCF Distribution instance.
```
Comment (gokarnm) [inline RELEASENOTES/v2.0.0.alpha-2.MD:22]:
```suggestion
Notation is a standalone client and is not integrated with Docker or Containerd clients. It uses a fixed configuration scheme in this release, where the certificates for signing and verifying are specified in a file. Refer https://github.com/notaryproject/notation/blob/main/specs/notation-config.md. It supports the operations for signing and verifying artifacts. This release depends on [oras-artifacts-spec](https://github.com/oras-project/artifacts-spec/) registry support for storing signatures. See [getting started](https://github.com/notaryproject/notation/blob/main/docs/hello-signing.md#getting-started) for using a local CNCF Distribution instance. 
```

**Review by gokarnm — CHANGES_REQUESTED** [review]

**Pull request merged**